### ISSUE TITLE
Docs our out of date from Latest Version 3.3.50

### ISSUE THREAD

**Issue by NinjaSudo** [wix/react-native-ui-lib#139]
Using `yarn add react-native-ui-lib` shows that I'm installing 3.3.50 and I'm also getting warnings with dependency linking:
```
warning " > babel-jest@22.2.2" has unmet peer dependency "babel-core@^6.0.0 || ^7.0.0-0".
warning " > react-native-ui-lib@3.3.50" has incorrect peer dependency "react@16.0.0".
warning " > react-native-ui-lib@3.3.50" has incorrect peer dependency "react-native@0.51.0".
```

**Comment by ethanshar** [comment]
I don't think the first warning comes from uilib since both babel dependencies are dev ones. 
regarding the other two warnings, I updated our `package.json` peer dependencies, please try to install again. (should be fixed on version 3.3.53)

**Comment by ethanshar** [comment]
Hey @NinjaSudo, is that fixed for you ?
